src/hoki.js

```diff
@@ -22,26 +20,14 @@ function handleRegisterType(type) {
 }
 
 // If the event is a string put it in an array and return
-function evtStrToArr(event) {
-    if (isArray(event)) {
-        return event;
-    }
-
-    event = [event];
-
-    return event;
-}
+const evtStrToArr = event => isArray(event) ? event : [event];
 
 // Register event(s)
 function register(event) {
     handleRegisterType(event);
 
     evtStrToArr(event).forEach(e => {
-        if (events[e]) {
-            return;
-        }
-
-        events[e] = [];
+        if (!events[e]) events[e] = [];
```

Comment:
And this is not to be an hardass, but I like the code style to be

if (!events[e]) {
    events[e] = [];
}

Not using one liners for if statements :)

Reply:
fair enough